crates/nu-command/src/debug/timeit.rs

```diff
@@ -44,14 +44,16 @@ impl Command for TimeIt {
         // Get the start time after all other computation has been done.
         let start_time = Instant::now();
 
+        // reset outdest, so the command can write to stdout and stderr.
+        let mut eval_stack = stack.clone().reset_out_dest();
         if let Some(command_to_run) = command_to_run {
             if let Some(block_id) = command_to_run.as_block() {
```

Comment:
```suggestion
            let stack = &mut stack.push_redirection(None, None);
            if let Some(block_id) = command_to_run.as_block() {
```

Reply:
I didn't document the stack redirection API too well, but a providing `None` to `push_redirection` will clear the corresponding pipe redirection if there is one. So, with this change, I don't think you need any of the other changes in this file. (E.g., this avoids the `Stack` clone.)

Reply:
Oh thanks!  After looking into documentation, `push_redirection` makes more sence to me